modular/manager/manager.go

```diff
@@ -42,6 +44,12 @@ type ManageModular struct {
 	scope   rcmgr.ResourceScope
 	mux     sync.Mutex
 
+	// loading task at startup.
+	enableLoadTask           bool
+	loadTaskLimitToReplicate int
```

Comment:
limit should use the queue cap.

Reply:
Yes, can't get cfg here and `manager.loadTaskLimitToReplicate = cfg.Parallel.GlobalReplicatePieceParallel`.